usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateFeedInfoEndDateAfterStartDate.java

```diff
@@ -61,7 +61,7 @@ public void execute() {
             final LocalDate feedStartDate = feedInfo.getFeedStartDate();
             final LocalDate feedEndDate = feedInfo.getFeedEndDate();
             if (feedStartDate != null && feedEndDate != null) {
-                if (feedStartDate.isBefore(feedEndDate)) {
+                if (!feedStartDate.isBefore(feedEndDate)) {
```

Comment:
Readability is improved by using non inverted logic
```suggestion
                if (feedEndDate.isBefore(feedStartDate)) {
```